UPGRADING.rst

```diff
@@ -10,6 +10,18 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#1272 Use Lambda layers to speed up \`make deploy\`
```

Comment:
```suggestion
#1272 Use Lambda layers to speed up ``make deploy``
```

No?

Reply:
Yes. Funny to witness that strange brain malfunction of mine.